spansy/src/http/span.rs

```diff
@@ -273,6 +281,27 @@ fn response_body_len(response: &Response) -> Result<usize, ParseError> {
     }
 }
 
+/// Parses a request or response body.
```

Comment:
I assume dechunking will be later called from inside this method, so we are passing in a `message body`.
```suggestion
/// Parses a request or response message body.
```